Source/Core/Core/IOS/ES/ES.h

```diff
@@ -228,6 +260,8 @@ class ES final : public Device
   IPCCommandResult DIGetTMDSize(const IOCtlVRequest& request);
   IPCCommandResult DIGetTMD(const IOCtlVRequest& request);
 
+  std::array<Context, 3>::iterator FindActiveContext(u32 fd);
```

Comment:
You can use a type alias here to avoid re-stating the same array bounds everywhere, i.e. under the Context struct you could do something like:

```cpp
// ES can only have 3 contexts at one time [Other necessary info]
using ContextArray = std::array<Context, 3>;
```

and then just use that.